test/vc-data-model-1.0/50-jwt.js

```diff
@@ -304,14 +294,65 @@ describe('JWT (optional)', function() {
     });
   });
 
-  describe('A verifiable presentation ...', function() {
-    it('vp MUST be present in a JWT verifiable presentation.', async function() {
-      const jwtBase64 = await util.generatePresentationJwt('example-016-jwt.jsonld', getGeneratorOptions(OPTIONS.JWT_PRESENTATION));
+  describe('A verifiable presentation ...', function() {    
+
+    it('vp MUST be present in a JWT verifiable presentation', async function() {
+      const jwtBase64 = await util.generatePresentationJwt('example-016-jwt-presentation.jsonld', getGeneratorOptions(OPTIONS.JWT_PRESENTATION));
       const jwtResult = cryptoFactory.constructJws(jwtBase64);
       expect(jwtResult.isContentWellFormedToken()).to.be.true;
 
       const payload = JSON.parse(jwtResult.getPayload());
       expect(payload.vp !== null && payload.vp !== undefined).to.be.true;
+      expect(payload.vp.type !== null && payload.vp !== undefined).to.be.true;
+      expect(payload.vp.verifiableCredential !== null && payload.vp !== undefined).to.be.true;
+    });    
+
+    it('aud MUST represent the subject of the consumer of the verifiable presentation', async function() {
+      const jwtBase64 = await util.generatePresentationJwt('example-016-jwt-presentation.jsonld', getGeneratorOptions(OPTIONS.JWT_PRESENTATION));
+      const jwtResult = cryptoFactory.constructJws(jwtBase64);
+      expect(jwtResult.isContentWellFormedToken()).to.be.true;
+
+      const payload = JSON.parse(jwtResult.getPayload());
+      expect(payload.aud !== null && payload.aud !== undefined).to.be.true;
+      expect(payload.aud).to.equal(aud);
+    });
+
+    it('jti MUST represent the id property of [...] the verifiable presentation', async function() {
+      const jwtBase64 = await util.generatePresentationJwt('example-016-jwt-presentation.jsonld', getGeneratorOptions(OPTIONS.JWT_PRESENTATION));
+      const jwtResult = cryptoFactory.constructJws(jwtBase64);
+      expect(jwtResult.isContentWellFormedToken()).to.be.true;
+
+      const payload = JSON.parse(jwtResult.getPayload());
+      expect(payload.jti !== null && payload.jti !== undefined).to.be.true;
+      expect(payload.jti).to.equal('urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5');
+    });
+
+    it('jti MUST represent the id property of [...] the verifiable presentation -- negative, no jti expected', async function() {
+      const jwtBase64 = await util.generatePresentationJwt('example-016-jwt-presentation-no-jti.jsonld', getGeneratorOptions(OPTIONS.JWT_PRESENTATION));
+      const jwtResult = cryptoFactory.constructJws(jwtBase64);
+      expect(jwtResult.isContentWellFormedToken()).to.be.true;
+
+      const payload = JSON.parse(jwtResult.getPayload());
+      expect(payload.jti === null || typeof payload.jti === 'undefined').to.be.true;
+    });
+        
+    it('iss MUST represent the issuer property of [...] the holder property of a verifiable presentation', async function() {
```

Comment:
because this test is only in the verifiable presentation section, it may be more clear for this instruction to be:
`iss MUST represent [...] the holder property of a verifiable presentation`

Reply:
i changed the language